test/e2e/live_iso_test.go

```diff
@@ -0,0 +1,45 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
+	"sigs.k8s.io/cluster-api/test/framework/clusterctl"
+)
+
+var _ = Describe("When testing live iso [live-iso]", func() {
+	liveIsoTest()
+})
+
+func liveIsoTest() {
+	It("Should create a target cluster nodes with live iso", func() {
+		controlPlaneMachineCount = int64(*e2eConfig.GetInt32PtrVariable("CONTROL_PLANE_MACHINE_COUNT"))
+		workerMachineCount = int64(*e2eConfig.GetInt32PtrVariable("WORKER_MACHINE_COUNT"))
+		result := &clusterctl.ApplyClusterTemplateAndWaitResult{}
+
+		clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
+			ClusterProxy: bootstrapClusterProxy,
+			ConfigCluster: clusterctl.ConfigClusterInput{
+				LogFolder:                clusterctlLogFolder,
+				ClusterctlConfigPath:     clusterctlConfigPath,
+				KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
+				InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
+				Flavor:                   "live-iso",
+				Namespace:                namespace,
+				ClusterName:              clusterName,
+				KubernetesVersion:        e2eConfig.GetVariable(KubernetesVersion),
+				ControlPlaneMachineCount: &controlPlaneMachineCount,
+				WorkerMachineCount:       &workerMachineCount,
+			},
+			CNIManifestPath:              e2eConfig.GetVariable(capi_e2e.CNIPath),
+			WaitForClusterIntervals:      e2eConfig.GetIntervals(specName, "wait-cluster"),
+			WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
+			WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
+		}, result)
+		targetCluster = bootstrapClusterProxy.GetWorkloadCluster(ctx, namespace, clusterName)
+		bootstrapClient := bootstrapClusterProxy.GetClient()
+		controlplaneM3Machines, workerM3Machines := getMetal3Machines(ctx, bootstrapClient, clusterName, namespace)
+		Expect(controlplaneM3Machines).To(HaveLen(int(controlPlaneMachineCount)))
+		Expect(workerM3Machines).To(HaveLen(int(workerMachineCount)))
+	})
```

Comment:
add check here by reading `/var/log/libvirt/qemu/"{{ node name }}-serial0.log` and verify it contains `"#  Welcome" `